scripts/barclamp_mgmt_lib.rb

```diff
@@ -67,6 +67,13 @@ def fatal(msg, log = nil, exit_code = 1)
   exit exit_code
 end
 
+def get_yml_paths_from_rpm(component)
+  rpm = "crowbar-#{component}"
+  get_rpm_file_list(rpm).select do |file|
+    file =~ %r!^#{CROWBAR_PATH}/barclamps/([^/]+).yml$!
```

Comment:
please make sure we don't include all the hound yml files here.
so basically exclude the dot yml files.
they are probably not shipped but for the dev setup this would be necessary